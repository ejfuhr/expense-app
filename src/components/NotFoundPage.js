import React from "react";
import { BrowserRouter, Route, Routes, Link, NavLink } from 'react-router-dom'
import Header from "./Header";

const NotFoundPage = () => (
    <div>
      This is your 404! <Link to="/">Go home </Link>
      
    </div>
);

export default NotFoundPage