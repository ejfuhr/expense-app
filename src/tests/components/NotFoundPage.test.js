import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';

test('should handle NotFoundPage correctly', () => {

    const wrapper = shallow(<NotFoundPage />)
    //expect(wrapper.find('<Link ').text()).toBe('Expensify');
    expect(wrapper).toMatchSnapshot();

})