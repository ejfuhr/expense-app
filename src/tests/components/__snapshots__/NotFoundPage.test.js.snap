// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should handle NotFoundPage correctly 1`] = `
<div>
  This is your 404! 
  <Link
    to="/"
  >
    Go home 
  </Link>
</div>
`;
